packages/vue-instantsearch/src/components/Hits.js

```diff
@@ -0,0 +1,57 @@
+import { createHits } from 'instantsearch-jsx';
+import { connectHitsWithInsights } from 'instantsearch.js/es/connectors';
+
+import { createSuitMixin } from '../mixins/suit';
+import { createWidgetMixin } from '../mixins/widget';
+import { renderCompat } from '../util/vue-compat';
+
+const augmentH = (baseH) => (tag, propsWithClassName, children) => {
+  const { className, ...props } = propsWithClassName;
+  return baseH(tag, Object.assign(props, { class: className }), [children]);
+};
+
+export default {
+  name: 'AisHits',
+  mixins: [
+    createWidgetMixin(
+      {
+        connector: connectHitsWithInsights,
+      },
+      {
+        $$widgetType: 'ais.hits',
+      }
+    ),
+    createSuitMixin({ name: 'Hits' }),
+  ],
+  props: {
+    escapeHTML: {
+      type: Boolean,
+      default: true,
+    },
+    transformItems: {
+      type: Function,
+      default: undefined,
+    },
+  },
+  computed: {
+    items() {
+      return this.state.hits;
+    },
+    widgetParams() {
+      return {
+        escapeHTML: this.escapeHTML,
+        transformItems: this.transformItems,
+      };
+    },
+  },
+  render: renderCompat(function (baseH) {
+    if (!this.state) return null;
+
+    const h = augmentH(baseH);
+
+    return createHits({ createElement: h })({
```

Comment:
Call to `createHits` could be externalized so that we don't call it every render

Reply:
how can it be externalised? if I remember it correctly in vue 2 you only have access to `h` in `render`'s arguments